src/main/kotlin/com/openlattice/data/storage/PostgresDataQueries.kt

```diff
@@ -68,7 +69,7 @@ fun buildPreparableFiltersSqlForLinkedEntities(
 
     val innerSql = selectEntitiesGroupedByIdAndPropertyTypeId(
             idsPresent = idsPresent, partitionsPresent = partitionsPresent, selectOriginIds = selectOriginIds
-    ) + " AND ${ORIGIN_ID.name} IS NOT NULL " + filtersClause + GROUP_BY_ESID_EKID_PART_PTID
+    ) + " AND ${ORIGIN_ID.name} IS NOT NULL AND ${ORIGIN_ID.name} IS NOT '${IdConstants.EMPTY_ORIGIN_ID.id}' " + filtersClause + GROUP_BY_ESID_EKID_PART_PTID
```

Comment:
`AND ${ORIGIN_ID.name} IS NOT NULL` seems removable

Reply:
Yes, given the doing-migration-to-non-null-and-setting-empty-uuid beforehand 👍